### ISSUE TITLE
OpenCV3?

### ISSUE THREAD

**Issue by pawelkomarnicki** [ruby-opencv/ruby-opencv#74]
It seems gem is for OpenCV2.4.12 — is there any way to use it with the newest OpenCV3? I tried face_detection and the "frontal face alt xml" file is in unsupported format.

**Comment by iredmedia** [comment]
Seems he's working on it according to #64.

**Comment by AntonTrapp** [comment]
I don't think so, look at the dates in the issue #64 . I think this gem seems deprecated and unmaintained (like about 99% of all gems *lol*). Using cpp version and calling system command from ruby now.

**Comment by iredmedia** [comment]
Sorry, I can't even read! Cold you post your solution?

**Comment by AntonTrapp** [comment]
Me? Not using this gem. The cpp version can be downloaded and compiled on Macs and Linux without any problem. Wrote the code in cpp, compiled it and calling it like any other system command.

**Comment by coderhs** [comment]
Is there any documentation on how to make the gem support the latest OpenCV? Would love to help out.

@ser1zw

**Comment by pducks32** [comment]
@coderhs I'd be happy to work with you on that goal. Maybe we can get this gem on its feet again?

**Comment by coderhs** [comment]
@pducks32 Sure, I would love to do that as well. But I never did actually figure out how to make the gem support the latest OpenCV. Hope to spend some time on it soon. If we have to bring this gem back to its feet, I think we should write a documentation on how to add new features. So that we can make more people involved in this.

**Comment by pducks32** [comment]
@coderhs We can always move this off of an Issue but I think it's best to start with tests and docs and then start implementing all of them. This library is a wrapper around the openCV C bindings so making sure we have tests and then making them pass seems best to me.

**Comment by JikkuJose** [comment]
I would also like to help!
